chapters/introduction.tex

```diff
@@ -30,94 +39,116 @@ \section{Doing credible research at scale}
-It is available as a free online collection of our resources and best practices.\sidenote{\url{http://dimewiki.worldbank.org/}}
-This book therefore complements the detailed-but-unstructured DIME Wiki
-with a guided tour of the major tasks that make up primary data collection.\sidenote{Like this: \url{https://dimewiki.worldbank.org/wiki/Primary_Data_Collection}}
+at the World Bank's \textbf{Development Economics (DEC) Vice Presidency}.\sidenote{
+\url{https://www.worldbank.org/en/about/unit/unit-dec}}
+
+DIME generates high-quality and operationally relevant data and research 
+to transform development policy, help reduce extreme poverty, and secure shared prosperity. 
+It develops customized data and evidence ecosystems to produce actionable information 
+and recommend specific policy pathways to maximize impact.
+DIME conducts research in 60 countries with 200 agencies, leveraging a 
+US\$180 million research budget to shape the design and implementation of 
+US\$18 billion in development finance. 
+DIME also provides advisory services to 30 multilateral and bilateral development agencies. 
+Finally, DIME invests in public goods to improve the quality and reproducibility of development research around the world. 
+
+DIME Analytics was created to take advantage of the concentration and scale of research at DIME to develop and test solutions, 
+to ensure high quality of data collection and research across the DIME portfolio, 
+and to make public training and tools available to the larger community of development researchers.
+Data for Development Impact compiles the ideas, best practices and software tools Analytics 
+has developed while supporting DIME's global impact evaluation portfolio. 
+The \textbf{DIME Wiki} is one of our flagship products, a free online collection of our resources and best practices.\sidenote{
+\url{http://dimewiki.worldbank.org/}}
+
+This book complements the DIME Wiki by providing a structured narrative of the data workflow for a typical research project. 
 We will not give a lot of highly specific details in this text,
-but we will point you to where they can be found,
-and give you a sense of what you need to find next.
-Each chapter will focus on one task,
-and give a primarily narrative account of:
+but we will point you to where they can be found.\sidenote{Like this: 
+\url{https://dimewiki.worldbank.org/wiki/Primary_Data_Collection}}
+Each chapter focuses on one task, providing a primarily narrative account of:
 what you will be doing; where in the workflow this task falls;
-when it should be done; who you will be working with;
-why this task is important; and how to implement the most basic form of the task.
-
-We will use broad terminology throughout this book
-to refer to different team members:
+when it should be done; and how to implement it according to best practices.
+
+
+
+\section{Outline of this book}
+The book progresses through the typical workflow of an empirical research project.
```

Comment:
```suggestion
The book progresses through the typical timeline of an empirical research project.
```

Is timeline better than workflow here? I want to make the distinction that we take the reader from beginning to end, and not just describe the workflow at a or a few points of a research project.

Reply:
i see what you're saying but i don't know if timeline is a good fit either. @bbdaniels @luizaandrade, thoughts?

Reply:
Progression? Stages?